crates/iota-graphql-rpc/src/extensions/logger.rs

```diff
@@ -199,10 +222,19 @@ impl Extension for LoggerExtension {
                         "[Schema] {}", resp.data
                     );
                 }
-                _ => info!(
+                _ if self.is_subscription.load(Ordering::Relaxed) => {
```

Comment:
Can't you take advantage of  the `operation_name`  here to resolve if the request is a subscription?

Reply:
Thank you for the clarifications!